loadtest/store/memstore/store.go

```diff
@@ -32,3 +39,15 @@ func (s *MemStore) SetUser(user *model.User) error {
 	s.user = user
 	return nil
 }
+
+func (s *MemStore) Post(postId string) (*model.Post, error) {
+	if post, ok := s.posts[postId]; ok {
+		return post, nil
+	}
+	return nil, fmt.Errorf("post with id %v not found", postId)
```

Comment:
I think the interface should have both return value and error, in case of another backend (not memory based) it would be good to know the reason for failure and differentiate it from "not found"

Reply:
Yes, the interface looks good. What I meant is that I probably wouldn't consider "not found" to be an error. If the function returns `nil, nil` then I know it wasn't found. If there's an actual error then it should return it.